### ISSUE TITLE
npm install error

### ISSUE THREAD

**Issue by ddtch** [awinogradov/generator-bem-ng#60]
Hello, I tried already many options including manual remove and install all dependencies but have an error with this. Any suggestions?
![image](https://cloud.githubusercontent.com/assets/765596/21463503/3188e970-c97b-11e6-95e8-805af87e49f2.png)

**Comment by awinogradov** [comment]
Hi! Have you tried to install bower-logger manually?

**Comment by ddtch** [comment]
@awinogradov yes, but it didn't help. Today I reinstalled node, npm, bower, yo.... installed bower-logger bower-npm-install... I really don't understand what I missed.
